### PR TITLE
[#363] Escape json using slim

### PR DESCRIPTION
**Pull request by ruslankhaertdinov**
Related to: https://github.com/fs/rails-base/issues/363

### CODE REVIEW

**Comment by ruslankhaertdinov** [comment]
@VladimirMikhailov please take a look.

File: spec/features/visitor/sign_in_spec.rb [hunk 1]
```diff
@@ -29,4 +30,10 @@ def sign_in(email, password)
 
     expect(page).to have_content("You have to confirm your email address before continuing.")
   end
+
+  scenario "js object stores Visitor data", js: true do
```
Comment (timurvafin) [inline spec/features/visitor/sign_in_spec.rb]: @ruslankhaertdinov I think this part of js should be tested somewhere else. May be we could use `jasmin` specs for that?

Comment (timurvafin) [inline spec/features/visitor/sign_in_spec.rb]: Ok, I found your discussion with @VladimirMikhailov in the  https://github.com/fs/rails-base/issues/363

Not sure current implementation with is readable. What if we just leave changes related to the escaping only?

Comment (ruslankhaertdinov) [inline spec/features/visitor/sign_in_spec.rb]: @timurvafin Thanks for the recommendation, I'll try to write jasmin tests for this.

Comment (ruslankhaertdinov) [inline spec/features/visitor/sign_in_spec.rb]: @timurvafin Do you mean leave changes without specs?

**Comment by ruslankhaertdinov** [comment]
@timurvafin @VladimirMikhailov What do you suggest: leave changes with current specs; remove specs and save only template changes or think about another js-testing utilities?

**Comment by VladimirMikhailov** [comment]
@ruslankhaertdinov hi, from my perspective, `= App.currentUserData = #{{current_user.to_json}};` looks more like a great example of how we can deal with preloaded data to interact later with it on the FE without making extra requests. And I can see the following options here : 

- Fix it without testing ( As @timurvafin has suggested )
- Extract the following logic from rails-base core to knowledge base(confluence) 
- Move the logic to example branch/repo if we have one and provide there better testing and use cases. For the instance we can create a case, when user has to see last sign in date in browser's time zone:

```ruby
scenario "Visitor sees the last sign in date" do
  expect(page).to have_last_sign_in_date_in(current_user, "MSC")
end
```

The first one "Fix it without testing" looks more reasonable for me, since it's more about the issue. Which one approach would be the best is absolutely up to you and @timurvafin. Thanks

**Comment by ruslankhaertdinov** [comment]
@VladimirMikhailov got it, thanks

**Comment by timurvafin** [comment]
@ruslankhaertdinov please cleanup commits.

**Comment by ruslankhaertdinov** [comment]
@timurvafin done

**Comment by timurvafin** [comment]
@ruslankhaertdinov I'm sorry, you need to rebase from master to let CI pass.

**Comment by ruslankhaertdinov** [comment]
@timurvafin no problem

**Pull request merged**